src/api/app/views/webui2/webui/package/show.html.haml

```diff
@@ -0,0 +1,67 @@
+:ruby
+  @pagetitle = "Show #{@project} / #{@package}"
+  @layouttype = 'custom'
+  devel_package = @package.develpackage
+= package_bread_crumb
+
+.card.mb-3
+  = render partial: 'tabs'
```

Comment:
For some reasons, my last comment was marked as outdated. So here it is again.

This causes an exception: https://obs-reviewlab.opensuse.org/davidkang-sprint44refactorp6-bootstrap-package-show/package/show/home:Admin/ruby

The fix: `= render(partial: 'tabs', locals: { project: @project, package: @package })`